src/devices/Board/AssemblyInfo.cs

```diff
@@ -0,0 +1,3 @@
+﻿using System.Runtime.CompilerServices;
+
+[assembly: InternalsVisibleTo("Board.Tests")]
```

Comment:
this is actually doable entirely from the csproj, see: https://github.com/dotnet/iot/blob/e3449040842ca153151cf6148aec133993b8d904/src/devices/Amg88xx/Amg88xx.csproj#L15-L20 (in pending PR)